man/systemd.network.xml

```diff
@@ -1903,6 +1912,19 @@
         <varname>DNS=</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>EmitPOP3Server=</varname></term>
+        <term><varname>POP3Server=</varname></term>
+
+        <listitem><para>Similar to the <varname>EmitDNS=</varname> and
+        <varname>DNS=</varname> settings described above, these
+        settings configure whether and what POP3 server information
+        shall be emitted as part of the DHCP lease. The same syntax,
+        propagation semantics and defaults apply as for
+        <varname>EmitDNS=</varname> and
+        <varname>DNS=</varname>.</para></listitem>
```

Comment:
Do we need `EmitPOP3Server=` at all? I'd say that if `POP3Server=` is configured, we should use it unconditionally.